app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="widget-wrapper">
+    <WidgetHead :config="getWidgetHeadConfig" />
+    <WidgetBody />
+    <WidgetFooter />
+  </div>
+</template>
+
+<script>
+import WidgetHead from './WidgetHead';
+import WidgetBody from './WidgetBody';
+import WidgetFooter from './WidgetFooter';
+
+export default {
+  name: 'Widget',
+  components: {
+    WidgetHead,
+    WidgetBody,
+    WidgetFooter,
+  },
+  props: {
+    welcomeHeading: {
+      type: String,
+      default: 'Hi There,',
+    },
+    welcomeTagLine: {
+      type: String,
+      default: '',
+    },
+    websiteName: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    websiteDomain: {
+      type: String,
+      default: '',
+    },
+    logo: {
+      type: String,
+      default: '',
+    },
+    isExpanded: {
+      type: Boolean,
+      default: true,
+    },
+    isOnline: {
+      type: Boolean,
+      default: true,
+    },
+    replyTime: {
+      type: String,
+      default: 'few minutes',
+    },
+  },
+  computed: {
+    // eslint-disable-next-line func-names
+    getWidgetHeadConfig: function() {
```

Comment:
Could use => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions to get rid of eslint errors.

Reply:
Done